.travis.yml

```diff
@@ -15,6 +15,8 @@ matrix:
   include:
     - rvm: 2.4.0
       env: COMMAND=rubocop
+before_install:
```

Comment:
Why was this not needed before but needed now?

Reply:
A change in travis I guess. I've removed this as it's sorted in https://github.com/zendesk/zendesk_apps_support/pull/269